elastic/kibana
# [Observability] [Exploratory View] exploratory view percentile breakdowns

## Summary

- Fixes #112860 Adds percentile breakdowns for:
   - KPI Over Time: UX (excluding page views)
   - KPI Over Time: Synthetics
<img width="1753" alt="Screen Shot 2021-10-02 at 9 32 39 AM" src="https://user-images.githubusercontent.com/11356435/135774004-2bac68f7-2616-4976-bf24-2aad04f9d048.png">
- Fixes #113712 Show legend always
<img width="1759" alt="Screen Shot 2021-10-03 at 6 46 11 PM" src="https://user-images.githubusercontent.com/11356435/135774003-08ab9f28-b851-4c9a-9d7f-10f64a77a990.png">
- Fixes #113327: Resolves bug with some labels missing for certain metrics when using percentile operation/breakdowns 


Testing
* Connect this branch to edge
* Navigate to exploratory view
* Select report type 'KPI Over Time'
* Select data type 'Synthetics'
* Select any metric
* Select the breakdown for this metric as percentile, and hit Apply
* Observe percentile breakdowns
* Repeat for all 'Synthetics' metric types
* Delete series
* Select data type 'RUM (UX)'
* Select any metric
* Select the breakdown for this metric as percentile, and hit Apply
* Observe percentile breakdowns
* Repeat for all 'UX' metric types, excluding Page Views. Ensure that page views metric does not have percentile breakdowns as an option
